rules/auth0_rules/auth0_user_invitation_created.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+from global_filter_auth0 import filter_include_event
+from panther_auth0_helpers import auth0_alert_context, is_auth0_config_event
+from panther_base_helpers import deep_get
+
+org_re = re.compile(r"^/api/v2/organizations/[^/\s]+/invitations$")
+
+
+def rule(event):
+    if not filter_include_event(event) or not is_auth0_config_event(event):
```

Comment:
negative conditionals compounded by `or` are more commonly represented by using the built-in `any()` 

```suggestion
    if not any([
       filter_include_event(event),
       is_auth0_config_event(event)
       ]):
```